src/SurveyDesign.jl

```diff
@@ -145,7 +169,38 @@ sampsize: [100, 100, 100  …  50]
 weights: [44.21, 44.21, 44.21  …  15.1]
 allprobs: [0.0226, 0.0226, 0.0226  …  0.0662]
 replicates: 1000
+
 ```
+
+If the replicate weights are given to us already, then we can directly pass them to the `ReplicateDesign` constructor. For instance, in
+the above example, suppose we had the `bootstrat` data as a CSV file.
+
+```jldoctest replicate-design
+julia> using CSV;
+
+julia> CSV.write("apistrat_withreplicates.csv", bootstrat.data);
+
+```
+
+We can now pass the replicate weights directly to the `ReplicateDesign` constructor.
+
+```jldoctest replicate-design
+julia> apistrat_fromcsv = CSV.read("apistrat_withreplicates.csv", DataFrame);
+
+julia> bootstrat_direct = ReplicateDesign(apistrat_fromcsv, [Symbol("replicate_"*string(replicate)) for replicate in 1:1000]; strata=:stype, weights=:pw)
+ReplicateDesign:
+data: 200×1044 DataFrame
+strata: stype
+    [E, E, E  …  H]
+cluster: none
+popsize: [4420.9999, 4420.9999, 4420.9999  …  755.0]
+sampsize: [100, 100, 100  …  50]
+weights: [44.21, 44.21, 44.21  …  15.1]
+allprobs: [0.0226, 0.0226, 0.0226  …  0.0662]
+replicates: 1000
+
+```
+
 """
 struct ReplicateDesign <: AbstractSurveyDesign
```

Comment:
I think there needs to be a `String` or `Symbol` argument of `type` of replicate design, eg "BRR", "subbootstrap", "jackknife". Its fine for now I think, but in future different types of replicate methods may diverge. @ayushpatnaikgit what do you think?

Reply:
Okay, I'll add the `type` fields. For now, will default it to `subbootstrap`.

Reply:
sounds cool